pkg/maps/lbmap/lbmap.go

```diff
@@ -622,8 +509,11 @@ func DumpServiceMapsToUserspace(includeMasterBackend bool) (loadbalancer.SVCMap,
 			idCache[fe.String()] = loadbalancer.ServiceID(k)
 		}
 
-		svc := newSVCMap.AddFEnBE(fe, be, svcKey.GetBackend())
-		newSVCList = append(newSVCList, svc)
+		// Do not include master services
+		if svcKey.GetBackend() != 0 {
```

Comment:
From what I reverse-engineered, it seems that the intention for the `includeMaster` flag was to to indicate that a service ID needs to be extracted when dumping the service maps. So, I didn't want skip the parsing and population bits to avoid loosing service ID during the restoration.

Reply:
That's about as far as I follow... but in the `SyncLBMap()` case we ignored master services previously. Including master services there wasn't explicitly called out in a commit message which is why I brought this up.

Reply:
Initially, I was afraid to miss a case when some old Cilium version set `revNATID` only for a master entry, but not for backend entries. However, I've just checked the commit history, and I couldn't find that it was ever actually the case. Going to change as you suggested.